print-radius.c

```diff
@@ -995,6 +1030,143 @@ print_attr_mip6_home_link_prefix(netdissect_options *ndo,
      nd_print_trunc(ndo);
 }
 
+static void
+print_attr_operator_name(netdissect_options *ndo,
+                    const u_char *data, u_int length, u_short attr_code _U_)
+{
+   u_int i;
+   u_int namespace_value;
+
+   ND_TCHECK_LEN(data, length);
+   if (length < 2)
+   {
+      ND_PRINT("ERROR: length %u < 2", length);
+      return;
+   }
+   namespace_value = EXTRACT_U_1(data);
+   data++;
+   ND_PRINT("[%s] ", tok2str(operator_name_vector, "Namespace %u not known", namespace_value));
+
+   for (i=0; i < length-1 && EXTRACT_U_1(data); i++, data++)
+      ND_PRINT("%c", ND_ASCII_ISPRINT(EXTRACT_U_1(data)) ? EXTRACT_U_1(data) : '.');
+
+   return;
+
+   trunc:
+      nd_print_trunc(ndo);
+}
+
+static void
+print_attr_location_information(netdissect_options *ndo,
+                    const u_char *data, u_int length, u_short attr_code _U_)
+{
+   u_int i;
+   u_int16_t index;
+   u_int8_t code, entity;
+   u_int64_t sighting_time, time_to_live;
+
+   ND_TCHECK_LEN(data, length);
+   if (length < 21)
+   {
+     ND_PRINT("ERROR: length %u < 21", length);
+      return;
+   }
+
+   index = EXTRACT_BE_U_2(data);
+   data += 2;
+
+   code = EXTRACT_U_1(data);
+   data++;
+
+   entity = EXTRACT_U_1(data);
+   data++;
+
+   sighting_time = EXTRACT_BE_U_8(data);
+   data += 8;
+
+   time_to_live = EXTRACT_BE_U_8(data);
+   data += 8;
+
+   ND_PRINT("Index %u, code %s, entity %s, sighting time %llu, time to live %llu: ",
```

Comment:
Use PRIu64 instead of %llu - that's the C99 way of printing a u_int64_t (which is *not* guaranteed to be an `unsigned long long int`):

    ND_PRINT("Index %u, code %s, entity %s, sighting time %llu, time to live %"PRIu64": ",